installer_scripts/common.sh

```diff
@@ -36,8 +36,10 @@ update_yb_migrate_bashrc() {
 
 install_yb_migrate() {
 	output "Installing yb_migrate."
-	$GO install github.com/yugabyte/yb-db-migration/yb_migrate@latest
+	pushd ../yb_migrate
+	$GO install
```

Comment:
any specific reason for this change? Both ways are fine, right?

Reply:
With the earlier command, the pre-commit tests were always installing `yb_migrate` from the `main` branch. What we actually need is to test the migration with the changes in the PR.

Reply:
Thanks.